pennylane/ops/qubit/special_unitary.py

```diff
@@ -298,32 +434,240 @@ def compute_matrix(theta, num_wires):
             A = qml.math.tensordot(theta, pauli_basis_matrices(num_wires), axes=[[-1], [0]])
         return qml.math.expm(1j * A)
 
-    @staticmethod
-    def compute_decomposition(theta, wires, num_wires):
-        r"""Representation of the operator as a product of other operators (static method).
+    def get_one_parameter_generators(self, interface=None):
+        r"""Compute the generators of one-parameter groups that reproduce
+        the partial derivatives of a special unitary gate.
 
-        .. math:: O = O_1 O_2 \dots O_n.
+        Args:
+            interface (str): The auto-differentiation framework to be used for the
+                computation.
```

Comment:
Mention what happend when `interface=None`.

Reply:
I added a "Raises:" section including this info, and mentioned the allowed interfaces.